test/hf/run_analysis.py

```diff
@@ -0,0 +1,760 @@
+'''Require: pip install torch transformers datasets wandb flaml[blendsearch,ray]
+'''
```

Comment:
I didn't review this file. Is it finalized or in progress? If in progress, you can exclude it from this PR.

Reply:
It's in progress, I'll exclude it before you are ready to approve